lucene/spatial3d/src/test/org/apache/lucene/spatial3d/geom/RandomGeo3dShapeGenerator.java

```diff
@@ -149,6 +150,26 @@ public PlanetModel randomPlanetModel() {
   }
 
   /**
+   * Method that returns a random generated Planet model as string from the supported
```

Comment:
I don't think you need this method.  I think you are using it only for init(map) when the test using it could instead simply do randomBoolean() to directly set the planet model on the factory.